src/game.js

```diff
@@ -0,0 +1,36 @@
+const Dictionary = require('./dictionary.js')
+
+class Game {
+    constructor() {        
+    }
+
+    static create(a, b) {
+        let keyId = Math.floor(Math.random() * 999999);
+        let game = {};
```

Comment:
it is better (for performance reasond) to create an object with all its attributes from the beginning when possible:
```javascript
const game = {
  id: keyId,
  hint:  "_ _ _ _ A",
  leftAttempts: 6,
  image: ` . . .  `
}
```